### ISSUE TITLE
Documentation Improvement Plans

### ISSUE THREAD

**Issue by tangy5** [Project-MONAI/MONAILabel#905]
**Description**
This ticket discusses the MONAILABEL documentation improvement plan. Please leave comment and suggestion on any part of the plan. 

**1.  More Intro to MONAILABEL Architecture**
Explain MONAILABEL workflow in three parts: 1.MONAILABEl server. 2. Client-end Software (3D Slicer): none MIA experts are not familiar with these tools.  3. Plugins.
Demonstrate the relationship between the server and visualization tool + plugins. 
Add a sample use case on what MONAILABEl can do at first look. 

- [x]  [Task 0 : Fine-tune the Modules Overview. Add more information to Introduction section in landing page. ]

**2. Improve the Installation Guide with Step-by-Step instructions on MONAILABEL**

Server—Client-end Software—Plugin
Separating quickstart page, move the 3D Slicer installation, plugin install to Installation section. 
Need to highlight instructions on Client Software and plugins, users may confused with 3D Slicer and Plugin installation.

- [x] [Task 1: Make a installation guide to 3D slicer (specify Slicer versions, Operating Systems)]

- [x] [Task 2: Make the detailed guide on how to install/load MONAILABEl plugins. (Finetune Current QuickStart)]

**3. Improve the Use Case Tutorial: Labeling Task | Bundle App**
Highlight labeling task. The multi-organ segmentation Radiology app can be demonstrated as the main function for labeling segmentation.
Prior tutorial?  
Need to make a comprehensive workflow on how labeling can be achieved. (Load, Inference, Save)
Add a tutorial on labeling any target with customized bundle.  

- [x] [Task 3: Make an easy-to-follow tutorial for Radiology app or a bundle app on how the labeling workflow performs.]

- [x] [Task 4: Make the tutorial for Bundle App on how MONAILABEL load customized DL models and labeling targets.] 

TO ADD MORE UPON DISCUSSION.

Thanks.

**Comment by Nic-Ma** [comment]
Cool, when introducing the MONAI Label server, maybe introduce the `sample apps` as unique section in it.

Thanks.

**Comment by zmsunnyday** [comment]
Great plan. Do we also plan to add docs of MONAILabel restful APIs? It will be very helpful for developers who want to develop a MONAILabel plugin. Although we have a page for APIs, it doesn't show all the information that can be carried in requests. And sometimes developers have to learn how to use them by trial and error. 

Thanks.

**Comment by diazandr3s** [comment]
> 

Thanks, @zmsunnyday  - this is a good point.
In the meantime, you could use this way to check the request nature via the browser. Please see this video:

https://user-images.githubusercontent.com/11991079/185203704-14f8c542-c5da-4b1c-ac0f-8c550bbe7ab8.mp4

**Comment by tangy5** [comment]
> Great plan. Do we also plan to add docs of MONAILabel restful APIs? It will be very helpful for developers who want to develop a MONAILabel plugin. Although we have a page for APIs, it doesn't show all the information that can be carried in requests. And sometimes developers have to learn how to use them by trial and error.
> 
> Thanks.

Thanks, this is a great point. Will discuss this in the following plan.
